test/model/store_test.dart

```diff
@@ -1082,6 +1082,22 @@ void main() {
         checkLastRequestFcm(token: '456def');
       }
     }));
+
+
+    test('fallback to default appBundleId in case packageInfo is null', ()async{
+      // This tests the case where packageInfo is null
+      // and default appBundleId gets passed to the [NotificationService.registerToken] method
+
+      prepareStore();
+      connection.prepare(json: {});
+
+      if(defaultTargetPlatform == TargetPlatform.iOS){
```

Comment:
This check doesn't seem right. Because the `debugDefaultTargetPlatformOverride` doesn't get updated, the block inside never gets executed. The other tests use this because they use `testAndroidIos` instead of `test`.

Reply:
nit:

```suggestion
      if (defaultTargetPlatform == TargetPlatform.iOS) {
```

Please also check for other spots for issues like this.

Reply:
I found these instructions in testAndroidIos function :
final origTargetPlatform = debugDefaultTargetPlatformOverride;
    addTearDown(() => debugDefaultTargetPlatformOverride = origTargetPlatform);
    debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
  
  what do you think of replicating them in the above test. It will look something like this:
  
  prepareStore();
      connection.prepare(json: {});

      final origTargetPlatform = debugDefaultTargetPlatformOverride;
      addTearDown(() {
        testBinding.reset();
        debugDefaultTargetPlatformOverride = origTargetPlatform;
      });

      debugDefaultTargetPlatformOverride = TargetPlatform.iOS;

      if (debugDefaultTargetPlatformOverride == TargetPlatform.iOS) {
        testBinding.packageInfoResult = null;
        await NotificationService.registerToken(connection, token: '012abc');
        checkLastRequestApns(token: '012abc', appid: 'com.zulip.flutter');
      }